vultr/resource_bare_metal.go

```diff
@@ -164,7 +164,7 @@ func resourceBareMetalRead(d *schema.ResourceData, meta interface{}) error {
 
 	instance, err := client.GetBareMetalServer(d.Id())
 	if err != nil {
-		if err.Error() == "Invalid bare metal instance." {
+		if err.Error() == "Invalid bare metal instance." || err.Error() == "Invalid server." {
```

Comment:
Dang, I really wish Vultr versioned the API and documented the codes/error messages for each endpoint. How about making `Invalid server.` the only value that is checked? I am testing the API manually with the example cURL commands given [0] and it looks like the error is always `Invalid server.` now.

```sh
curl -H "API-Key: $VULTR_API_KEY" https://api.vultr.com/v1/baremetal/list?SUBID=foo
```

[0] https://www.vultr.com/api/#baremetal_baremetal_list